pkg/controller/forwarding/forwarding_controller.go

```diff
@@ -98,9 +98,12 @@ func (r *ReconcileForwarding) Reconcile(request reconcile.Request) (reconcile.Re
 	//check for instancename and then update status
 	var reconcileErr error = nil
 	if instance.Name != singletonName {
-		instance.Status = logforwarding.NewForwardingStatus(logforwarding.LogForwardingStateRejected, logforwarding.LogForwardingReasonName, singletonMessage)
+		instance.Status = logforwarding.NewForwardingStatus()
+		instance.Status.State = logforwarding.LogForwardingStateRejected
+		instance.Status.Reason = logforwarding.LogForwardingReasonName
+		instance.Status.Message = singletonMessage
 	} else {
-		instance.Status = logforwarding.NewForwardingStatus(logforwarding.LogForwardingStateAccepted, logforwarding.LogForwardingReasonName, "")
+		instance.Status = logforwarding.NewForwardingStatus()
```

Comment:
since it looks like this pattern is used again, maybe we can create a different function that just returns a default StateAccepted status

Reply:
reverted to be more like a proper constructor